packages/picasso.js/src/core/chart-components/ref-line/refline.js

```diff
@@ -265,6 +288,7 @@ const refLineComponent = {
           p,
```

Comment:
We can remove p and use only slopeLine which can cover a normal ref-line.

Reply:
Fixed